setup.py

```diff
@@ -0,0 +1,45 @@
+from setuptools import find_packages, setup
+
+requirements = ["pycryptodome>=3.0.0", "eth-abi>=4.0.0"]
+
+pydantic_requirements = ["pydantic>=2.0.0"]
+
+tests_requirements = ["pytest-cov>=4.0.0", "pytest>=7.0.0"]
+
+all_requirements = pydantic_requirements
+
+
+VERSION = "0.1.0"
+DESCRIPTION = "MerkleTree implementation"
+LONG_DESCRIPTION = "Yet another package for MerkleTree, this time compatible with OpenZeppelin implementation"
+
+setup(
+    name="merkle-zeppelin",
+    version=VERSION,
+    description=DESCRIPTION,
+    long_description=LONG_DESCRIPTION,
+    author="akcelero",
+    author_email="akcelero@gmail.com",
+    license="MIT",
+    packages=find_packages(),
+    install_requires=requirements,
+    tests_requires=tests_requirements,
+    keywords="merkle tree merkletree openzeppelin",
+    classifiers=[
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3 :: Only",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
You use shortland of `typing.Union`, which is available since python 3.10 only.
[Docs on using |](https://docs.python.org/3/library/typing.html#typing.Union)
```suggestion
```

Reply:
good idea, I will use `Union` in order make version 3.8 and 3.9 works
thanks : )